### PR TITLE
nikolai/replace the banner

### PR DESCRIPTION
**Pull request by nikolai-deriv**

### CODE REVIEW

File: templates/partials/bbot-to-deriv-banner.mustache [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="dbot-banner eu-hide invisible">
```
Comment (sara-fs) [inline templates/partials/bbot-to-deriv-banner.mustache:1]: Actually there is a MovingBanner component which is react based and you can use it

File: src/common/utils/utility.js [hunk 2]
```diff
@@ -64,3 +62,9 @@ export const moveToDeriv = async () => {
         window.location.replace('https://binary.com/move-to-deriv');
     }
 };
+
+export const getClientsCountryByIP = async () => {
```
Comment (sara-fs) [inline src/common/utils/utility.js:66]: We have saved the country on localStorage. I think you can remove these codes and use the storage.

File: templates/partials/bbot-to-deriv-banner.mustache (hunk 1 above)
Comment (nikolai-deriv) [reply templates/partials/bbot-to-deriv-banner.mustache:1]: thank you, i've seen it, but decided to copy the "try DBot" banner

File: src/common/utils/utility.js (hunk 2 above)
Comment (nikolai-deriv) [reply src/common/utils/utility.js:66]: unfortunately country in localStroage is only available for logged clients, and i need to check it for non-logged clients as well

**Review by mehrzad-fs — DISMISSED** [review]

**Review by mehrzad-fs — APPROVED** [review]

**Pull request merged**